code/campaignclassic/src/main/res/layout/push_template_filmstrip_carousel.xml

```diff
@@ -76,16 +95,18 @@
 
     <ImageButton
         android:id="@+id/leftImageButton"
-        android:layout_width="30dp"
-        android:layout_height="40dp"
-        android:layout_marginTop="130dp"
+        android:layout_width="15dp"
+        android:layout_height="20dp"
+        android:layout_centerVertical="true"
+        android:layout_alignParentLeft="true"
         android:background="@drawable/skipleft" />
 
     <ImageButton
         android:id="@+id/rightImageButton"
-        android:layout_width="30dp"
-        android:layout_height="40dp"
-        android:layout_marginTop="130dp"
+        android:layout_width="15dp"
+        android:layout_height="20dp"
+        android:layout_centerVertical="true"
+        android:layout_centerInParent="true"
```

Comment:
Do we need `layout_centerInParent`? That would also align the button in the center horizontally. Since we already have `layout_alignEnd` this would do nothing

Reply:
you are correct, this isn't needed